doc/source/getting_started/drivers.rst

```diff
@@ -29,7 +29,7 @@ Java
 - `Astyanax <https://github.com/Netflix/astyanax/wiki/Getting-Started>`__
 - `Casser <https://github.com/noorq/casser>`__
 - `Datastax Java driver <https://github.com/datastax/java-driver>`__
-- `Kundera <https://github.com/impetus-opensource/Kundera>`__
+- `Kundera <https://github.com/impetus/opensource/Kundera>`__
```

Comment:
https://github.com/impetus/Kundera should be the correct link.

https://github.com/impetus/opensource/Kundera leads to 404 also

Reply:
:smile:  ok that was sad, thanks for correcting me!